dev/io.openliberty.microprofile.openapi.2.0.internal_fat/publish/servers/OpenAPIMergeTestServer/jvm.options

```diff
@@ -0,0 +1 @@
+-Dcom.ibm.ws.beta.edition=false
```

Comment:
Do we need this? It should be false by default, right?

Reply:
It put in in there because we're explicitly depending the value being false but its not actually needed.

Reply:
Ok, makes sense.